source/tutorials/thermal-daemon/thermal-daemon.rst

```diff
@@ -0,0 +1,66 @@
+.. _thermal-daemon:
+
+Thermal Daemon: Intel thermal solution for |C|
+==============================================
+
+Introduction
+------------
+
+`Thermal daemon (TD) <https://github.com/intel/thermal_daemon>`_ is an opensource thermal solution provided by Intel. It is a service that gets started at Android filesystem bootup and executes throughout the uptime of the system. It supports multiple thermal policies such as active, passive and critical, users can configure thermal policies for a platform using the *thermal-conf.xml* in the */vendor/etc/thermal_daemon* directory on the target.
```

Comment:
Suggested rewrite: 
`Thermal daemon (TD) <https://github.com/intel/thermal_daemon>`_ is an open source thermal solution provided by Intel. It is a service that starts at the Android filesystem boot up and executes throughout the uptime of the system. It supports multiple thermal policies such as active, passive, and critical. Users can configure thermal policies for a platform using *thermal-conf.xml* in the */vendor/etc/thermal_daemon* directory on the target.

Reply:
thanks